services/workflows-service/src/ui-definition/utils/collection-flow/9-process-details.ts

```diff
@@ -24,47 +24,60 @@ const validationSchema = {
                         'monthlySalesVolume',
                         'monthlyTransactions',
                         'averageTicketAmount',
+                        'mainCategory',
+                        'businessModel',
                       ],
-                      // errorMessage: {
-                      //   minLength: {
-                      //     monthlySalesVolume: 'Monthly Sales Volume is required.',
-                      //     monthlyTransactions: 'Monthly Transactions are required.',
-                      //     averageTicketAmount: 'Average Ticket Amount is required.',
-                      //     mainCategory: 'Main Category is required.',
-                      //     businessModel: 'Business Model is required.',
-                      //   }
-                      // },
-                      // errorMessage: {
-                      //   minLength: {
-                      //     monthlySalesVolume: 'Monthly Sales Volume is required.',
-                      //     monthlyTransactions: 'Monthly Transactions are required.',
-                      //     averageTicketAmount: 'Average Ticket Amount is required.',
-                      //     mainCategory: 'Main Category is required.',
-                      //     businessModel: 'Business Model is required.',
-                      //     isSpikeInSales: 'Is Spike In Sales is required.',
-                      //     spikeSalesAverageVolume: 'Spike Sales Average Volume is required.',
-                      //     spikeTransactionNumber: 'Spike Transaction Number is required.',
-                      //     spikeOfVolumeInRegion: 'Spike Of Volume In Region is required.',
-                      //   }
-                      // },
+                      errorMessage: {
```

Comment:
The structure is not clear, error message (singular) has multiple required string fields?